samples/js/node/notebooks/001-hello-world.nnb

```diff
@@ -117,15 +145,15 @@
         {
             "language": "javascript",
             "source": [
-                "const imagenetClassesMap = require('../../assets/imagenet_classes_map.json');\n\nconsole.log(`Result: ${imagenetClassesMap[resultIndex]}`);"
+                "const imagenetClassesMap = require('../../assets/datasets/imagenet_class_index.json');\n\nconsole.log(`Result: ${imagenetClassesMap[resultIndex][1]}`);"
             ],
             "outputs": [
                 {
                     "items": [
                         {
                             "mime": "application/vnd.code.notebook.stdout",
                             "value": [
-                                "Result: flat-coated retriever",
+                                "Result: curly-coated_retriever",
```

Comment:
Please make sure we are using the same imagenet_classes 
"flat-coated retriever" has 208 index when curly-coated is 209